lib/puppet/face/module/uninstall.rb

```diff
@@ -69,9 +69,8 @@
         exit 1
       else
         mod = return_value[:affected_modules].first
-        "Removed '#{return_value[:module_name]}'" <<
-        (mod.version ? " (#{colorize(:cyan, mod.version.to_s.sub(/^(?=\d)/, 'v'))})" : '') <<
-        " from #{mod.modulepath}"
+        module_version = mod.version ? "(#{colorize(:cyan, mod.version.to_s.sub(/^(?=\d)/, 'v'))})" : ''
+        _("Removed '%{name}' %{module_version} from %{path}") % { name: return_value[:module_name], module_version: module_version, path: mod.modulepath }
```

Comment:
Thanks for cleaning this up.

One minor change - I the previous lines accommodated the extra space in the dynamically added content so that regardless of how it came together there weren't extraneous spaces between words. Can you do the same?